packages/mux-player/src/index.ts

```diff
@@ -1221,6 +1224,27 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     }
   }
 
+  get extraPlaylistParams() {
+    // NOTE: the official type definition for the 0th `URLSearchParams` constructor argument is stricter than what is actually
+    // allowed and commonly used (e.g. a key with a boolean or number value). Ignoring to allow these cases. (CJP)
+    // @ts-ignore
+    if (!this.hasAttribute(PlayerAttributes.EXTRA_PLAYLIST_PARAMS)) return DEFAULT_EXTRA_PLAYLIST_PARAMS;
+    return [
+      ...new URLSearchParams(this.getAttribute(PlayerAttributes.EXTRA_PLAYLIST_PARAMS) as string).entries(),
```

Comment:
NOTE: Uses `URLSearchParams` constructor to more easily/reliably parse the attribute string, which acts as the "source of truth" for the value.